src/render/Texture.js

```diff
@@ -107,12 +136,29 @@ define([
          * @param {DrawContext} dc The current draw context.
          */
         Texture.prototype.bind = function (dc) {
-            dc.currentGlContext.bindTexture(dc.currentGlContext.TEXTURE_2D, this.textureId);
+            var gl = dc.currentGlContext;
+
+            gl.bindTexture(gl.TEXTURE_2D, this.textureId);
+
+            this.applyTexParameters(dc);
+
             dc.frameStatistics.incrementTextureLoadCount(1);
             return true;
         };
 
         /**
+         * Applies the configured texture parameters to the OpenGL context.
+         * @param {DrawContext} dc The current draw context.
+         */
+        Texture.prototype.applyTexParameters = function (dc) {
+            var gl = dc.currentGlContext;
+
+            // Configure the OpenGL texture magnification function. Use linear by default.
```

Comment:
For the problem at hand, #243, changing only the magnification filter gives the best results in my opinion. Should we also consider the minification filter in this update? There is quite an added complexity to it due to the dependency on the logic introduced by the wrap mode in the constructor.